dist/grafana/grafana-prometheus-zrepl.json

```diff
@@ -249,7 +249,7 @@
             "stack": true,
             "steppedLine": false,
             "targets": [{
-                "expr": "zrepl_version_daemon{job='$prom_job_name'}",
+                "expr": "sgn(zrepl_uptime{job='$prom_job_name'})",
```

Comment:
Re your question about backwards-compatible panel in Grafana (https://github.com/zrepl/zrepl/issues/553#issuecomment-1011207474)
I wouldn't bother with it, as long as we mention it in the changelog.

Reply:
Okay.